### PR TITLE
docs: known issue of tb_plugin

### PR DESCRIPTION
**Pull request by jgongd**
## Description

Pytorch Profiler TensorBoard Plugin is not compatible with our latest environment image, because it has been [deprecated](https://pytorch.org/tutorials/intermediate/tensorboard_profiler_tutorial.html#use-tensorboard-to-view-results-and-analyze-model-performance) by PyTorch. Add it to the known issue.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/release-notes/pytorch-tensorboard-plugin.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:orphan:
+
+**Known Issue**
+
+-  PyTorch has `deprecated
+   <https://pytorch.org/tutorials/intermediate/tensorboard_profiler_tutorial.html#use-tensorboard-to-view-results-and-analyze-model-performance>`
+   their Profiler TensorBoard Plugin. Our latest environment image comes with PyTorch 2.3, so some
+   of the tb_plugin features may not work. If you are looking to use all the features, we suggest
```
Comment (azhou-determined) [inline docs/release-notes/pytorch-tensorboard-plugin.rst:8]: would put tb_plugin in double backticks, \``tb_plugin\``

**Review by azhou-determined — APPROVED** [review]
thanks for finding this bug and seeing it through 🔥

File: docs/release-notes/pytorch-tensorboard-plugin.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+:orphan:
+
+**Known Issue**
+
+-  PyTorch has `deprecated
+   <https://pytorch.org/tutorials/intermediate/tensorboard_profiler_tutorial.html#use-tensorboard-to-view-results-and-analyze-model-performance>`
+   their Profiler TensorBoard Plugin (``tb_plugin``), so some features may not be compatible with
+   PyTorch 2.0 and above. Our current default environment image comes with PyTorch 2.3. If users
+   experiencing issues with this plugin, we suggest using an image with PyTorch version earilier
```
Comment (mackrorysd) [inline docs/release-notes/pytorch-tensorboard-plugin.rst:9]: earilier -> earlier, if we get a chance to revise this before release

Comment (jgongd) [reply docs/release-notes/pytorch-tensorboard-plugin.rst:9]: Good catch! Revised: https://github.com/determined-ai/determined/pull/10183

**Pull request merged**